overrides/macos-override.json

```diff
@@ -865,6 +865,15 @@
                     "state": "enabled"
                 }
             }
+        },
```

Comment:
Do we want this on macos?

Reply:
@SabrinaTardio  asked for it. https://app.asana.com/0/0/1206753149050909/1207007243483505/f